src/webview/devfile-registry/registryViewLoader.ts

```diff
@@ -42,6 +43,15 @@ async function devfileRegistryViewerMessageListener(event: any): Promise<any> {
         case 'openInBrowser':
             vscode.commands.executeCommand('openshift.componentType.openStarterProjectRepository', starterProject);
             break;
+        case 'telemeteryCopyEvent':
+            const devFileName = event.devFile;
+            const copyClicked = event.copyClicked;
+            const telemetryEventCopyDevFile = new ExtCommandTelemetryEvent('openshift.registryView.starterProjects.copyDevFile');
+            telemetryEventCopyDevFile.send({
+                devFileName: devFileName,
+                devFileCopied: copyClicked
```

Comment:
As I said above command's id 'openshift.registryView.starterProjects.copyDevFile' means exactly the same.